src/Canvas2d/babylon.bounding2d.ts

```diff
@@ -0,0 +1,169 @@
+﻿module BABYLON {
+
+    /**
+     * Stores 2D Bounding Information.
+     * This class handles a circle area and a bounding rectangle one.
+     */
+    export class BoundingInfo2D {
+
+        /**
+         * The coordinate of the center of the bounding info
+         */
+        public center: Vector2;
+
+        /**
+         * The radius of the bounding circle, from the center of the bounded object
+         */
+        public radius: number;
+
+        /**
+         * The extent of the bounding rectangle, from the center of the bounded object.
+         * This is an absolute value in both X and Y of the vector which describe the right/top corner of the rectangle, you can easily reconstruct the whole rectangle by negating X &| Y.
+         */
+        public extent: Vector2;
+
+        constructor() {
+            this.radius = 0;
+            this.center = Vector2.Zero();
+            this.extent = Vector2.Zero();
+        }
+
+        public static ConstructFromSize(size: Size): BoundingInfo2D {

```

Comment:
We use Create for similar APIs so can you please rename ConstructXXXX by CreateXXXX